app/views/notifications_mailer/weekly_student_summary.html.erb

```diff
@@ -69,28 +69,28 @@
     <p>Here is what you should focus on right now.</p>
 
 <%  if @top_tasks && @top_tasks.count > 0 %>
-<%      if @overdue_top && @overdue_top.count > 0 %>
-    <p>Catch up by completing the following <b>overdue</b> task<%="s" if @overdue_top.count > 1%>!</p>
+<%      if @overdue_top && @overdue_top.count > 0 %> #@summarytask.each do |sm| @summarytask[:due](sm)
```

Comment:
Same for all of the comments in this file. Please take care with what you are putting forward to be merged.